src/App.tsx

```diff
@@ -1,25 +1,74 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
-
-import './App.scss';
-import Start from './components/Start';
-import { Finish } from './components/Finish';
-
-import { isLoading, getMessage } from './store';
+import { useSelector, useDispatch } from 'react-redux';
+import { getPreparedTodos } from './api';
+import { TodoList } from './components/ToDoList';
+import './App.css';
 
+import {
+  isLoading,
+  getSortedTodos,
+  setTodos,
+  setError,
+  startLoading,
+  finishLoading,
+  sortTodo,
+  loadSortButtons,
+} from './store';
 
 const App = () => {
   const loading = useSelector(isLoading);
-  const message = useSelector(getMessage) || 'Ready!';
+  const todos = useSelector(getSortedTodos);
+  const loadButtons = useSelector(loadSortButtons);
+  const dispatch = useDispatch();
+
+  const downloadTodos = async () => {
+    dispatch(startLoading());
+    try {
+      const data = await getPreparedTodos();
+
+      dispatch(setTodos(data));
+    } catch (error) {
+      dispatch(setError(`ERROR: ${error}`));
+    }
+
+    dispatch(finishLoading());
+  };
+
+  const sortByTitle = () => {
+    dispatch(sortTodo('title'));
```

Comment:
create one handler for sorting and pass this string via data-attribute or something like this